source/extensions/tracers/zipkin/zipkin_tracer_impl.h

```diff
@@ -158,12 +171,11 @@ class ReporterImpl : public Reporter, Http::AsyncClient::Callbacks {
    *
    * @param driver ZipkinDriver to be associated with the reporter.
    * @param dispatcher Controls the timer used to flush buffered spans.
-   * @param collector_endpoint String representing the Zipkin endpoint to be used
+   * @param CollectorInfo represents the Zipkin endpoint information to be used.
```

Comment:
Not sure Info is idiomatic in general when talking about config. If so, ignore this.

```suggestion
   * @param CollectorInfo represents the Zipkin configuration to be used.
```